src/components/capabilities/implementation/fission-lobby.ts

```diff
@@ -289,27 +294,20 @@ async function translateClassifiedInfo(
 async function retry<T>(
   action: () => Promise<T>,
   options: { tries: number; timeout: number; timeoutMessage: string }
-): Promise<T | null> {
+): Promise<T> {
   return await Promise.race([
-    (async () => {
-      let returnValue
-      let tryNum = 1
-      while (tryNum <= options.tries) {
-        try {
-          returnValue = await action()
-        } catch (e) {
-          if (tryNum == options.tries) {
-            throw e
-          }
-        }
-        tryNum++
-      }
-      return returnValue || null
-    })(),
```

Comment:
This was loading the iframe 20 times at the same time instead of waiting for the timeout.